engine/gctscript.go

```diff
@@ -0,0 +1,82 @@
+package engine
+
+import (
+	"fmt"
+	"path/filepath"
+	"sync/atomic"
+
+	"github.com/thrasher-corp/gocryptotrader/gctscript/vm"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+const gctscriptManagerName = "GCTScript"
+
+type gctScriptManager struct {
+	running  atomic.Value
+	shutdown chan struct{}
+}
+
+// Started returns if gctscript manager subsystem is started
+func (g *gctScriptManager) Started() bool {
```

Comment:
Please link this up to GetSubsystem and SetSubsystem (https://github.com/thrasher-corp/gocryptotrader/blob/master/engine/helpers.go#L36). Moving forward, this just gave me the idea to have a RegisterSubsystem handler which will automatically add this functionality without us having to specifically hardcore it